rest_framework/pagination.py

```diff
@@ -36,10 +36,14 @@ def _positive_int(integer_string, strict=False, cutoff=None):
 
 def _divide_with_ceil(a, b):
     """
-    Returns 'a' divded by 'b', with any remainder rounded up.
+    Returns 'a' divided by 'b', with any remainder rounded up.
     """
+    if a == 0 or b == 0:
```

Comment:
Yeah, something like that.
I'd probably do...

  if self.limit:
      ... # determine current/final
  else:
     # set default values for current/final